automation/src/test/scala/org/broadinstitute/dsde/firecloud/component/Clickable.scala

```diff
@@ -25,14 +28,19 @@ trait Clickable extends LazyLogging { this: Component =>
     try {
       click on query
     } catch {
+      // on rare occasion, encounters stale exception
+      case e: StaleElementReferenceException => click on query
```

Comment:
happened once during my watch. making sure it won't happen ever again in future.